### ISSUE TITLE
Improve logging/output of pipeline

### ISSUE THREAD

**Issue by wbenoit26** [ML4GW/aframev2#260]
The logging that comes out of law/luigi is confusing for new users. It's quite verbose, and when something goes wrong, it can be hard to tell what happened (or, if it happened in a condor job, where to look to see what went wrong). What can we do to improve the way logging is done?

**Comment by wbenoit26** [comment]
If it's not possible to turn down the number of messages from `law`, then one thing I was thinking of was trying to be more explicit about what's going on using our own logging. I've noticed myself slipping on the amount of logging statements I have, just because I've already got a good sense of what's going on. Obviously there's a limit to how much we want to output, but I think we could add more messages indicating what's happening at the current moment (and make it clear that the message is coming from Aframe, rather than from law)

**Comment by EthanMarx** [comment]
Yeah very true - need to figure out how to get a hold of the logger for logging to console from tasks.

For condor jobs theres not much we can do - users can inspect the condor job logs themselves. We can add logging to the depoyment task however
